sdk/pubkey/src/lib.rs

```diff
@@ -4,6 +4,17 @@ pub use five8_const::decode_32_const;
 pub use pinocchio;
 
 /// Convenience macro to define a static `Pubkey` value.
+/// 
+/// This macro validates the pubkey at compile time, ensuring it's a valid
```

Comment:
Not always in compile time – it depends on the input. The `const` qualifier allows it to be done in compile time when the input is static.